project/scenes/game/characters/player/States/State.gd

```diff
@@ -1,24 +1,27 @@
 class_name State extends Node
 
 #("Character States")
-var idle_state : State
-var walking_state : State
-var air_state : State
-var dashing_state : State
-var climbing_state : State
-var casting_state : State
-var dead_state : State
+var idle_state: State
+var walking_state: State
+var air_state: State
+var attack_state: State
+var dashing_state: State
+var climbing_state: State
+var casting_state: State
+var dead_state: State
 
 var player
 var input_check
 var animation_tree
-
+var CSM
+var variables
```

Comment:
**variables** ne pomeni nič, rabi bolj descriptive imena, tudi če so bolj dolga (in dodaj type)